faiss-search/scann/scann.py

```diff
@@ -49,6 +50,7 @@ def __init__(self,
             If this number is increased, the accuracy will increase but it will impact speed
         """
         super().__init__(*args, **kwargs)
+        self.searcher = None
```

Comment:
This is normally so, but to avoid `jina` from serializing this object, this initialization should be avoided.